duckduckgo/Android
# Fix popup styles in dark theme

Task/Issue URL: https://app.asana.com/0/276630244458377/1200804933122285
Tech Design URL: 
CC: 

**Description**:
This PR fixes a bug where the popup menu items show the wrong text color when in dark mode

**Steps to test this PR**:
1. Set the theme to dark
1. Go to the bookmarks activity and check all the popup menu items have the correct color
1. Go to the tan switcher activity and check all the popup menu items have the correct color


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
